### ISSUE TITLE
State of CSS 2022: No translation found for at least one of those intlKeys

### ISSUE THREAD

**Issue by Romjkez** [StateOfJS/StateOfJS-Vulcan#157]
Happens at: https://survey.devographics.com/survey/state-of-css/2022/read-only
Got there from https://survey.devographics.com/survey/state-of-css/2022 -> Review Survey

**Problem**
Look at the questions titles and console warnings
![Capture](https://user-images.githubusercontent.com/36418657/200192681-8142170f-4188-4b5d-9122-d3088e71ad66.PNG)

**Expected**
Translations load correctly and can be read

p.s if you click "Shapes & Graphics" button at the bottom of the page, the app routes us to the same page with different URL https://survey.devographics.com/survey/state-of-css/2022/undefined/1 and the translations are now correct, but "undefined" in the URL seems like a bug

**Comment by SachaG** [comment]
You're right, there's some issues with the "read-only" mode for surveys that are already closed, it seems. I'll look into it.
